test/references/DataCite Ecology.txt

```diff
@@ -1,20 +1,12 @@
 register(DataDep(
-    "Data from Ecology and genomics of an important crop wild relative as a prelude to agricultural innovation",
+    "Data from: Ecology and genomics of an important crop wild relative as a prelude to agricultural innovation",
```

Comment:
That is not right.
A DataDep name can not contain the `:` character.
As it has to be a valid filepath.
Something has gone wrong somewhere with the `shortname` stuff